nixos/modules/virtualisation/digital-ocean-image.nix

```diff
@@ -0,0 +1,58 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.virtualisation.digitalOceanImage;
+  defaultConfigFile = pkgs.writeText "configuration.nix" ''
+    { ... }:
+    {
+      imports = [
+        <nixpkgs/nixos/modules/virtualisation/digital-ocean-image.nix>
```

Comment:
We could use modules path here but I'm impartial about it. See previous comment

Reply:
Fixed to use modulesPath as I do think it is better (ref above)